### PR TITLE
Resolves #1224

### LINKED ISSUE #1224: Example 2 is misleading
The second example has a bizarre output.  The default output should be as follows.  

An error occurred on the server when processing the URL. Please contact the system administrator. &lt;p/&gt; If you are the system administrator please click &lt;a href="https://g
o.microsoft.com/fwlink/?LinkID=82731"&gt;here&lt;/a&gt; to find out more about this error.

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: 51cdca0a-de14-9ef4-6488-404968dcb273
* Version Independent ID: 0a08acf0-78bb-1d77-7f60-91e2244e8296
* Content: [Get-IISConfigAttributeValue (iisadministration)](https://docs.microsoft.com/en-us/powershell/module/iisadministration/get-iisconfigattributevalue?view=win10-ps)
* Content Source: [docset/windows/iisadministration/get-iisconfigattributevalue.md](https://github.com/MicrosoftDocs/windows-powershell-docs/blob/master/docset/windows/iisadministration/get-iisconfigattributevalue.md)
* Product: **w10**
* Technology: **powershell-windows**
* GitHub Login: @coreyp-at-msft
* Microsoft Alias: **brianlic**

### PR DESCRIPTION
**Pull request by velkovb**
Fix Example 2 output.
https://github.com/MicrosoftDocs/windows-powershell-docs/issues/1224

### CODE REVIEW

**Comment by e0i** [comment]
@JohanFreelancer9 Copy editing is needed for this PR. Thanks.

**Comment by e0i** [comment]
@dariomws306 @get-itips @tseward

Peer review needed for this PR. Thanks.

File: docset/windows/iisadministration/get-iisconfigattributevalue.md [hunk 1]
```diff
@@ -48,7 +48,7 @@ This command gets the configuration attribute MaxUrlSegments for the Default Web
 ### Example 2: Get configuration attribute values from different parts of the configuration store
 ```
 PS C:\> Get-IISConfigSection "system.webServer/asp" | Get-IISConfigAttributeValue -AttributeName "ScriptErrorMessage"
-error message inside the location path default web site
+An error occurred on the server when processing the URL. Please contact the system administrator. <p/> If you are the system administrator please click <a href="http://go.microsoft.com/fwlink/?LinkID=82731">here</a> to find out more about this error.
```
Comment (JohanFreelancer9) [inline docset/windows/iisadministration/get-iisconfigattributevalue.md]:
```suggestion
An error occurred on the server while processing the URL. Please contact the system administrator. <p/> If you are the system administrator, please click <a href="http://go.microsoft.com/fwlink/?LinkID=82731">here</a> to find out more about this error.
```

Comment (velkovb) [reply docset/windows/iisadministration/get-iisconfigattributevalue.md]: This message is actually returned by the command so I have copied it and pasted in the article. If we want to edit it we need to edit the code of IIS I guess.

**Comment by JohanFreelancer9** [comment]
My apologies. Please ignore my edit.

**Comment by e0i** [comment]
The PR is good to go.

**Pull request merged**